app/src/main/res/drawable/progress_bar.xml

```diff
@@ -4,7 +4,9 @@
     <shape>
       <corners android:radius="8dp" />
```

Comment:
It should be `6dp`. I think in drawable files it file if follow the exact dimens as per mocks  rather than multiple of 4. Mostly if the value is less than 10dp. It still is debatable and therefore depends on your choice but I think its fine to have 6dp in this case.

Reply:
Through out app we are following this multiple of four unless its shadow, more than that I am changing the height of progress bar to 12dp.

Reply:
Okay sounds good. Thanks